crypto/x509v3/v3_ncons.c

```diff
@@ -435,6 +436,27 @@ int NAME_CONSTRAINTS_check_CN(X509 *x, NAME_CONSTRAINTS *nc)
     return X509_V_OK;
 }
 
+/*
+ * Return nonzero if the GeneralSubtree has valid 'minimum' field
+ * (must be absent or 0) and valid 'maximum' field (must be absent).
+ */
+static int nc_minmax_valid(GENERAL_SUBTREE *sub) {
+    BIGNUM *bn = NULL;
+    int ok = 1;
+
+    if (sub->maximum)
+        ok = 0;
```

Comment:
Why not `return 0` directly?

Reply:
I changed back to `ok = 0` for consistency with how the `minimum` field is checked (it sets `ok = 0` so that the bignum can be freed before returning).

Reply:
IMO, it's not necessary to keep consistent with the `minimum` checking here. But I don't think this would a blocker for this PR, so either `return 0` or `ok = 0` sounds OK to me...